integrations/notion/src/actions/add-comment-to-discussion.ts

```diff
@@ -0,0 +1,16 @@
+import * as notion from '../notion'
+import type { IntegrationProps } from '.botpress'
+
+export const addCommentToDiscussion: IntegrationProps['actions']['addCommentToDiscussion'] = async ({ ctx, input }) => {
+  try {
+    const response = await notion.addCommentToDiscussion(ctx, input.discussionId, input.commentBody)
+    if (response) {
+      console.info('Successfully added comment to discussion')
+      return { success: true, response }
```

Comment:
**nitpick:** 
1. these fields (`sucess`, `response`) are currently not declared in your definition so they will be removed from the backend 
2. I don't believe we should return them; no other integration does it. I think you should remove the try-catch statemen, let your code fail and let the SDK will handle the rest

(same goes for all other actions)